transformations/geonames_transformation/transformation.py

```diff
@@ -0,0 +1,272 @@
+import io
+import os
+import nltk
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class GeoNamesTransformation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
```

Comment:
Please remove task "TEXT_TAGGING". This transformation is adding new words to a given sentence, so we can't use this for text tagging unless the generation method produces the respective tagging sequence along with the generated transformation.

Reply:
You are right, the method is adding new words. "TEXT_TAGGING" is now removed.